File: Java/Kode Program/Individu.java
public class Individu {
    protected String noKTP;
    protected String nama;
    protected String alamat;

    public Individu() {}

    public Individu(String noKTP, String nama, String alamat) {
        this.noKTP = noKTP;
        this.nama = nama;
        this.alamat = alamat;
    }

    public void setNoKTP(String ktp) { this.noKTP = ktp; }
    public void setNama(String n) { this.nama = n; }
    public void setAlamat(String a) { this.alamat = a; }

    public String getNoKTP() { return noKTP; }
    public String getNama() { return nama; }
    public String getAlamat() { return alamat; }

    public void printInfo() {
        System.out.println("KTP: " + noKTP + ", Nama: " + nama + ", Alamat: " + alamat);
    }
}

File: Python/Kode Program/Individu.py
class Individu:
    def __init__(self, noKTP="", nama="", alamat=""):
        self.noKTP = noKTP
        self.nama = nama
        self.alamat = alamat

    def setNoKTP(self, ktp): self.noKTP = ktp
    def setNama(self, n): self.nama = n
    def setAlamat(self, a): self.alamat = a

    def getNoKTP(self): return self.noKTP
    def getNama(self): return self.nama
    def getAlamat(self): return self.alamat

    def printInfo(self):
        print(f"KTP: {self.noKTP}, Nama: {self.nama}, Alamat: {self.alamat}")
